.github/workflows/release-artifacts.yaml

```diff
@@ -0,0 +1,43 @@
+name: Release Artifacts
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.14
+      id: go
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: go get -t -v -d ./...
+
+    - name: Get tag
+      uses: olegtarasov/get-tag@v2
+      id: tagName
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Build on PR
+      id: docker_build
+      uses: docker/build-push-action@v2
+      with:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is this secret stored in the common place already?

Reply:
This is generated automatically by github

Reply:
aaaaah i see we're pushing to github packages as opposed to docker hub.
worth adding a note of this in the Readme?

Reply:
I will add it along the installation note when we have an artifact to use :)